astro.config.mjs

```diff
@@ -10,5 +10,6 @@ export default defineConfig({
     // Some old docs may still point to .html urls for terms and privacy.
     '/terms.html': '/terms',
     '/privacy.html': '/privacy',
+    '/pricing': '/#pricing',
```

Comment:
I tried this, it didn't work.

Reply:
yeah I reverted...I thought it was working for a second but I guess it was cached.